rofi-power-menu

```diff
@@ -13,7 +13,7 @@ set -e
 set -u
 
 # All supported choices
-all=(shutdown reboot suspend hibernate logout lockscreen)
+all=(shutdown reboot suspend hibernate logout lockscreen windows)
 
 # By default, show all (i.e., just copy the array)
 show=("${all[@]}")
```

Comment:
```suggestion
# By default, show all others except "Reboot to Windows"
show=(shutdown reboot suspend hibernate logout lockscreen)
```

Reply:
Yes, makes sense. Then this windows option should get added to the readme for visibility.